clients/privacy-center/public/fides-js-demo.html

```diff
@@ -17,8 +17,11 @@
         window.render_delay = 0; // increment the delay to test/mimic a slow loading or dynamic page
         const params = new URLSearchParams(window.location.search);
         const geolocation = params.get("geolocation");
+        const gpp = params.get("gpp");
         const src = `/fides.js${!!params.size ? `?${params.toString()}` : ""}`;
-
+        if (gpp === "debug") {
+          document.write('<script src="/fides-ext-gpp.js"><\/script>');
+        }
```

Comment:
when `gpp=debug` we're loading the js file instead of including it as part of the fides.js payload. That way we can benefit from the SourceMaps :)